src/main/java/org/candlepin/hostedtest/HostedTestCloudRegistrationAdapter.java

```diff
@@ -78,4 +179,18 @@ public CloudAccountData setupCloudAccountOrg(String cloudAccountID, String cloud
         return new CloudAccountData("owner_key", false);
     }
 
+    private boolean isOwnerEntitledToProduct(String ownerKey, String productId) {
+        if (ownerKey == null || productId == null) {
+            return false;
+        }
+
+        Set<String> subIds = this.datastore.productSubscriptionMap.getOrDefault(productId, new HashSet<>());
```

Comment:
micro-optimization: the new `HashSet<>()` can be changed to Set.of() for fewer characters, or `Collections.emptySet()` can be used to avoid the overhead of creating the new instance entirely.

Reply:
Updated